### PR TITLE
Update keycloak-httpd-client-install.8

### PR DESCRIPTION
**Pull request by theinrich88**
corrects templates path + typo

### CODE REVIEW

**Comment by theinrich88** [comment]
Any problem ?

File: doc/keycloak-httpd-client-install.8 [hunk 1]
```diff
@@ -68,7 +68,7 @@ disable. Or the pathname to a OpenSSL CA bundle to use.
 .TP
 .BR \-\-template\-dir " " \fITEMPLATE_DIR\fR
 Template location
-(default: /usr/share/keycloak\-httpd\-client/templates)
+ (default: /usr/share/keycloak\-httpd\-client\-install/templates)
```
Comment (thalman) [inline doc/keycloak-httpd-client-install.8:71]: Is this leading space intentional?

Comment (thalman) [reply doc/keycloak-httpd-client-install.8:71]: This is not crucial, resolving

**Comment by thalman** [comment]
late but still, Thanks

**Pull request merged**